src/map/skill.cpp

```diff
@@ -5107,6 +5107,19 @@ int skill_castend_damage_id (struct block_list* src, struct block_list *bl, uint
 	map_freeblock_lock();
 
 	switch(skill_id) {
+	case HFLI_MOON:	//[orn]
+	case HFLI_SBR44:	//[orn]
+	{
+#ifdef RENEWAL
+		if (hd != nullptr)
+		{
+			skill_blockhomun_start(hd, skill_id, skill_get_cooldown(skill_id, skill_lv));
```

Comment:
Not all homunculus abilities work like this where the cooldown is defined in the cooldown line, sometimes they are in gettime2, and are not changed depending on whether the homunculus has a master or not, in the prerenewal homunculus S they continue to have a cooldown instead of time2, I don't think that be right, but i can clean some block_pc from code by putting it on blockhomun

Reply:
solved on [Daegaladh](https://github.com/Daegaladh) update